packages/block-editor/src/components/index.js

```diff
@@ -45,6 +45,7 @@ export { default as ToolSelector } from './tool-selector';
 export { default as URLInput } from './url-input';
 export { default as URLInputButton } from './url-input/button';
 export { default as URLPopover } from './url-popover';
+export { __experimentalImageURLInputUI } from './url-popover/image-url-input-ui';
```

Comment:
Do we need to include the `ImageURLInputUI` change here or can we move it into a separate PR?

Reply:
I think this was because the branch needed a rebase.